src/Data/Unrestricted/Linear/Internal/Instances.hs

```diff
@@ -23,13 +23,8 @@ module Data.Unrestricted.Linear.Internal.Instances where
 
```

Comment:
Maybe it would make sense to rename this module to `OtherInstances` or `ExtraInstances` given that most of the instances were moved back into `Consumable` and `Dupable`?

Reply:
I would like to wait till another PR to deal with the rest of that module. My general preference would be to get rid of it altogether if that can be done without too much pain.

Reply:
I agree. I also plan on doing some reorganisation of the `Control.Functor` internals in another PR